.github/workflows/feature-request.yml

```diff
@@ -26,8 +24,9 @@ jobs:
         if: github.event_name != 'issues' || contains(github.event.issue.labels.*.name, 'feature-request')
         uses: ./actions/feature-request
         with:
-          appInsightsKey: ${{secrets.TRIAGE_ACTIONS_APP_INSIGHTS}}
           token: ${{secrets.VSCODE_ISSUE_TRIAGE_BOT_PAT}}
+          repo: "vscode"
+          owner: "microsoft"
```

Comment:
Could you use the [GITHUB_REPOSITORY](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) variable for this?

Reply:
No. The [GITHUB_REPOSITORY](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) in this case could be `vscode-engineering` since the workflow is run in that context.